app/src/userpages/components/DashboardPage/List/index.jsx

```diff
@@ -26,6 +27,8 @@ import useFilterSort from '$userpages/hooks/useFilterSort'
 
 import NoDashboardsView from './NoDashboards'
 
+const generateTimeAgoDescription = (updatedDate: Date) => moment(updatedDate).fromNow()
```

Comment:
Should we consolidate how we do this  (from the StreamList page)?

```
const timezone = moment.tz.guess()
const nowTime = moment.tz(Date.now(), timezone)
moment.min(moment.tz(stream.lastUpdated, timezone), nowTime).fromNow()
```

Reply:
There's also `import { ago } from '$shared/utils/time'` which works without `moment`. I thought we got rid of `moment` already but seems it's still there 🤷‍♂

Reply:
Good points, `ago` might be the best. I'll make a new issue about it.